umm-spec-lib/resources/json-schemas/collection/umm/v1.0/umm-search-results-json-schema.json

```diff
@@ -64,9 +62,54 @@
           "description": "The mime type of the original metadata",
           "type": "string",
           "minLength": 1
+        },
```

Comment:
I feel like having this in versions 1.0 through 1.8 defeats the purpose of a versioned schema. Is there a particular reason for adding it to all versions?

Reply:
First of all, this schema is used only in our testing code. Secondly, as we are not really versioning the search result schema in CMR, rather just link to different versions of UMM-C schema here, there is no way to add new fields in the search results without updating all the old schemas.

Reply:
Which is another point backing up your question as to the usefulness of having this versioned ...

Reply:
The reason this is versioned is not for versioning the fields in the meta field, but for pointing to the versioned UMM-C schema. I am OK with this as it is only for testing, but we should probably remove the search result schema from the EMFD repo. I would like to know if that is OK.